<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="639" started-at="2021-01-17T14:51:12 EET" finished-at="2021-01-17T14:51:13 EET">
    <groups>
    </groups>
    <test name="Default test" duration-ms="639" started-at="2021-01-17T14:51:12 EET" finished-at="2021-01-17T14:51:13 EET">
      <class name="packNew.TemplateFrameWork.LoginTest">
        <test-method status="PASS" signature="startDriver()[pri:0, instance:packNew.TemplateFrameWork.LoginTest@3b088d51]" name="startDriver" is-config="true" duration-ms="13415" started-at="2021-01-17T14:50:59 EET" finished-at="2021-01-17T14:51:12 EET">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startDriver -->
        <test-method status="PASS" signature="openLoginScreen()[pri:0, instance:packNew.TemplateFrameWork.LoginTest@3b088d51]" name="openLoginScreen" is-config="true" duration-ms="1" started-at="2021-01-17T14:51:12 EET" finished-at="2021-01-17T14:51:12 EET">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openLoginScreen -->
        <test-method status="FAIL" signature="TestLoginMethod()[pri:0, instance:packNew.TemplateFrameWork.LoginTest@3b088d51]" name="TestLoginMethod" duration-ms="15" started-at="2021-01-17T14:51:12 EET" finished-at="2021-01-17T14:51:12 EET">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [hello world] but found [hello]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [hello world] but found [hello]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at packNew.TemplateFrameWork.LoginTest.TestLoginMethod(LoginTest.java:24)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestLoginMethod -->
        <test-method status="PASS" signature="takeScreenShotWhenFailure(org.testng.ITestResult)[pri:0, instance:packNew.TemplateFrameWork.LoginTest@3b088d51]" name="takeScreenShotWhenFailure" is-config="true" duration-ms="567" started-at="2021-01-17T14:51:12 EET" finished-at="2021-01-17T14:51:13 EET">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=TestLoginMethod status=FAILURE method=LoginTest.TestLoginMethod()[pri:0, instance:packNew.TemplateFrameWork.LoginTest@3b088d51] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenShotWhenFailure -->
        <test-method status="PASS" signature="endDriver()[pri:0, instance:packNew.TemplateFrameWork.LoginTest@3b088d51]" name="endDriver" is-config="true" duration-ms="1" started-at="2021-01-17T14:51:13 EET" finished-at="2021-01-17T14:51:13 EET">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endDriver -->
      </class> <!-- packNew.TemplateFrameWork.LoginTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
